scout/adapter/mongo/panel.py

```diff
@@ -522,9 +522,11 @@ def apply_pending(self, panel_obj, version):
 
     def latest_panels(self, institute_id, include_hidden=False):
         """Return the latest version of each panel."""
-        panel_names = self.gene_panels(
-            institute_id=institute_id, include_hidden=include_hidden
-        ).distinct("panel_name")
+        panel_names = sorted(
```

Comment:
Mmm shouldn't show in panel page?

I see this:

<img width="379" alt="image" src="https://user-images.githubusercontent.com/28093618/160108922-09a6fd56-fdf3-49c8-9fe0-2b6de1786c85.png">

Reply:
Should and does locally! Let me check..

Reply:
It's on stage

Reply:
Looks good to me:
![Screenshot 2022-03-25 at 12 04 34](https://user-images.githubusercontent.com/758570/160110109-a8fd64b7-d5f2-469d-aa32-1300c797cbab.png)
But yeah, maybe there is something odd with the PanelApp panels!

Reply:
Great job, thanks! Will this alphabetical order also be reflected in the drop-down list on the SNV/INDELs variants page?

Reply:
Maybe it would also be more obvious if I sort on display name instead!

Reply:
> Great job, thanks! Will this alphabetical order also be reflected in the drop-down list on the SNV/INDELs variants page?

Hi, np! No, not immediately; for us that is a very short list, and kind of sectioned into the ones from the case, and the institute own additions. But, maybe I could add like a separator between those and sort each. I can give it a quick look.

Reply:
Oh. Well, for us it would certainly be very helpful to get the panels alphabetically also there.

Reply:
Ok, like this for the panelS page: now with display name instead of panel name as well, which should make it better.
![Screenshot 2022-03-25 at 12 32 55](https://user-images.githubusercontent.com/758570/160113527-4e35771f-66eb-4530-80cb-73cd3f12fd30.png)

Reply:
> Looks good to me:

Perhaps I was still on the old branch? Yes it works!